src/DataFrames.jl

```diff
@@ -9,14 +9,23 @@ module DataFrames
 
 using Reexport, StatsBase, SortingAlgorithms, Compat
 @reexport using CategoricalArrays, Missings
-using Base: Sort, Order
+using Base.Sort
+using Base.Order
 using Compat.Unicode
 using Compat.Printf
 
 if VERSION >= v"0.7.0-DEV.2738"
+    using Unicode: normalize
+
     const kwpairs = pairs
 else
     kwpairs(x::AbstractArray) = (first(v) => last(v) for v in x)
+    macro isdefined(sym)
+        sym = Expr(:quote, sym)
+        esc(:(isdefined($sym)))
+    end
+    using Compat.IOBuffer
+    const normalize = normalize_string  # requires that we don't use LinearAlgebra normalize
```

Comment:
I'd rather wrap this inside `module Unicode`, and avoid `using Unicode: normalize` in the other branch, so that you write `Unicode.normalize` everywhere to avoid the ambiguity with `LinearAlgebra.normalize`.

Reply:
will do...

Reply:
Julia is not letting me define a module in a conditional statement.  Any other ideas for dealing with this?  I really want to try hard to put only valid 0.7 code in the main body.